include/grpcpp/security/tls_credentials_options.h

```diff
@@ -56,7 +56,7 @@ class TlsKeyMaterialsConfig {
   int version() const { return version_; }
 
   /** Setter for key materials that will be called by the user. The setter
-   * transfers ownership of the arguments to the config. **/
+   *  takes ownership of the arguments to the config. **/
```

Comment:
Please revise this comment saying that the ownership will be transferred.

Reply:
Per our offline discussion, this comment is no longer needed.